evmbin/src/main.rs

```diff
@@ -14,7 +14,21 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Parity EVM interpreter binary.
+//! Parity EVM Interpreter Binary.
+//!
+//! ## Overview
+//!
+//! The Parity EVM interpreter binary is an additional tool in the Parity
+//! Ethereum toolchain. It is an EVM implementation for Parity Ethereum that
+//! may be used to run a standalone version of the EVM interpreter.
+//!
+//! ## Usage
+//!
+//! Build Parity Ethereum from source, then start it with:
+//! ```bash
+//! cd evmbin && cargo build -p evmbin && cd ..
+//! ./target/debug/parity-evm --help
```

Comment:
If built with `--release` the path becomes `./target/release/parity-evm` (and we might want to suggest they copy the executable to, dunno, `~/bin`?).

Reply:
installing binaries is usually out of scope here

Reply:
If they're using macOS or Linux and they want it in their PATH, we could suggest that they create a symlink i.e. `sudo ln -s <ABSOLUTE_PATH>/parity-ethereum/target/release/parity-evm /usr/local/bin`, assuming that `echo $PATH` indicates that /usr/local/bin is in their path, and then they can run `parity-evm --help` from any directory.
I'd have to look into what steps are required for Windows.

Reply:
Or should we just add a general suggestion like: _"Optionally if you want to be able to just run `parity-evm` from any directory then either copy the executable file itself to a directory that is in your PATH variable, or just create a symlink."_

Reply:
Personally I'd avoid symlinks to the build directory because files therein are clobbered anytime I build. Maybe a user of `evmbin` doesn't build as often as I do, but who knows. I think the wording could just be: "if you wish to run parity-evm from any directory copy the executable from ./target/release to a directory in your $PATH, e.g. ~/bin or /usr/local/bin.".

Reply:
Thanks, i've pushed a commit with this change

Reply:
again, I would like to emphasize that explaining users how to use binaries is out of scope for our documentation.

Reply:
@soc1c I've pushed another commit removing the explanation about how to use binaries